.claude/agents/flutter-performance-analyzer.md

```
@@ -0,0 +1,528 @@
+- Memory profile may differ from Skia - Impeller uses different GPU memory patterns
+- RepaintBoundary behavior may change - Impeller handles repaints differently
+- If you still see first-frame jank, it's likely widget building or data loading, not shaders
+
+## Using Flutter DevTools
+
+### Launch DevTools
+
+```bash
+# Method 1: From running app
+flutter run
+# Press 'v' to open DevTools in browser
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)